Makefile

```diff
@@ -76,14 +76,19 @@ format: $(GOIMPORTS)
 test: ## Runs all Go unit tests.
 	@echo ">> running tests for all modules: $(MODULES)"
 	for dir in $(MODULES) ; do \
-		cd $${dir} && go test -v -race ./...; \
+		cd $${dir} && go test -v -race $(go list ./... | grep -v examples/exampleapp); \
```

Comment:
Could we use a system in Go code to skip these tests by default? I was wondering [the short tests approach](https://pkg.go.dev/testing#hdr-Skipping) or an env var. This way it becomes more friendly to some IDE/text editors tooling too.

Reply:
We were discussing @jessicalins at length how to do this best, we did not want to add environment variable to check, since we wanted to leave the test code as minimal as possible for demonstration purposes. The `short` test approach however looks interesting :thinking:, although this test is not really long or short, the problem is it is intended to be manually interacted with, so it cannot run in CI. @jessicalins WDYT?

Reply:
Okay, sounds good to me. I just wanted to keep the project friendly to running `go test ./...` on the root folder. It can be improved later. 👍